templates/repo/issue/view_content/pull.tmpl

```diff
@@ -372,8 +374,8 @@
 				</div>
 			{{end}}
 
-			{{if and .ShowMergeInstructions .Issue.PullRequest.HeadRepo (not .Issue.PullRequest.HasMerged) (not .Issue.IsClosed)}}
-				{{template "repo/issue/view_content/pull_merge_instruction" .Issue.PullRequest}}
+			{{if and .Issue.PullRequest.HeadRepo (not .Issue.PullRequest.HasMerged) (not .Issue.IsClosed)}}
+				{{template "repo/issue/view_content/pull_merge_instruction" dict "PR" .Issue.PullRequest "ShowMergeInstructions" .ShowMergeInstructions}}
```

Comment:
I would prefer a longer name like "PullRequest" instead of "PR" (although it's not a blocker)